File: task-03/Go/prime_numbers_upto_n.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Println("Enter a number:")
	fmt.Scan(&n)

	var i, j int
	var flag bool

	fmt.Println("Prime numbers up to", n, "are:")

	for i = 2; i <= n; i++ {
		flag = false

		for j = 2; j <= i/2; j++ {
			if i%j == 0 {
				flag = true
				break
			}
		}

		if flag == false {
			fmt.Println(i)
		}
	}
}

File: task-03/JavaScript/prime_numbers_upto_n.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter a number: ', (n) => {
  const number = parseInt(n);

  if (!isNaN(number)) {
    let primes = [];

    for (let i = 2; i <= number; i++) {
      let isPrime = true;
      for (let j = 2; j * j <= i; j++) {
        if (i % j === 0) {
          isPrime = false;
          break;
        }
      }
      if (isPrime) {
        primes.push(i);
      }
    }

    console.log("Prime numbers up to " + number + " are: " + primes.join(", "));
  } else {
    console.log("Invalid input. Please enter a valid number.");
  }

  rl.close();
});
